docs/guidance/ingress.md

```diff
@@ -70,6 +73,61 @@ kubectl describe ingress raycluster-ingress-head-ingress
 #               will report "404 Not Found".
 ```
 
+### Example: Manually setting up NGINX Ingress on KinD
+```sh
+# Step 1: Create a KinD cluster with `extraPortMappings` and `node-labels`
+cat <<EOF | kind create cluster --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+  extraPortMappings:
+  - containerPort: 80
+    hostPort: 80
+    protocol: TCP
+  - containerPort: 443
+    hostPort: 443
+    protocol: TCP
+EOF
+
+# Step 2: Install NGINX ingress controller
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
+kubectl wait --namespace ingress-nginx \
+  --for=condition=ready pod \
+  --selector=app.kubernetes.io/component=controller \
+  --timeout=90s
+
+# Step 3: Install KubeRay operator
+pushd helm-chart/kuberay-operator
+helm install kuberay-operator .
+
+# Step 4: Install RayCluster and create an ingress separately. 
+popd
```

Comment:
Please teach users how to modify the YAML file both in this doc and `ray-cluster.separate-ingress.yaml`. For example, "...backend.service.name" should be the same as head service.